File: 02_functions.java
class Functions{

    public static String getString(){
        return "100pts";
    }

    public static int getNumbers(){
        return 100;
    }

    public static boolean getBoolean(){
        return false;
    }
    public static void main(String[] args) {
        System.out.println(getString());
        System.out.println(getNumbers());
        System.out.println(getBoolean());
        System.out.println(((Object) getString()).getClass().getSimpleName());
        System.out.println(((Object) getNumbers()).getClass().getSimpleName());
        System.out.println(((Object) getBoolean()).getClass().getSimpleName());
    }
}

File: 02_functions.dart
void main()
{
    print(getNumbers());
    print(getStrings());
    print(getBoolean());
    
    print(getNumbers().runtimeType); //like in java non-primitives start with small letter //int, bool
    print(getStrings().runtimeType); //like in java primitives start with Capital letter //String
    print(getBoolean().runtimeType);
}

String getStrings() //like in java dont need to use public static but use capital S for string
{
  return "100pts";
}

int getNumbers()
{
  return 100;
}

bool getBoolean()
{
  return false;
}